source/_patterns/02-molecules/leads/lead-gallery.twig

```diff
@@ -2,9 +2,18 @@
 
   <div class="c-lead-gallery__top">
 
-    {% include '@molecules/article/figure-with-caption.twig' %}
+    <div class="c-lead-gallery__image js-toggle js-gallery-toggle" data-toggled="c-main" data-prefix="gallery">
+      {%
+        include 'molecules-figure'
+        with {
+          caption: false,
+          credit: false
+        }
+      %}
+    </div>
 
     <div class="c-lead-gallery__text">
+      {% include 'molecules-credit' %}
       <h2 class="c-lead-gallery__title">{{ lead_slides_title }}</h2>
       <div class="c-lead-gallery__dek">
         <p>{{ lead_slides_dek }}</p>
```

Comment:
visually speaking, this is where the `caption` would go

Reply:
I usually put the credit inside the `<figcaption>`. From what I've seen, this seems to be the standard (mostly because NYTimes does it). I'm open to putting it wherever though. As long as it's semantic and accessible.